rmc/resources/rmc.py

```diff
@@ -624,19 +613,6 @@ def gnomad_fitted_score_path(
 )
 """
 Table containing missense variants in canonical transcripts annotated with MPC.
-"""
-
-mpc_release_dedup = VersionedTableResource(
-    default_version=CURRENT_FREEZE,
-    versions={
-        freeze: TableResource(
-            path=f"{MPC_PREFIX}/{CURRENT_GNOMAD_VERSION}/{freeze}/mpc_dedup.ht"
-        )
-        for freeze in FREEZES
-    },
-)
-"""
-Table containing missense variants in canonical transcripts annotated with MPC.
 
 This Table contains only one row per each unique locus/alleles combination.
 """
```

Comment:
There seems to be a bug in line 623 with variable `oe_bin_counts_tsv` where `{CURRENT_FREEZE}` is written twice and really should be `{CURRENT_GNOMAD_VERSION}` ?

Reply:
good catch!